recipes/vmtouch/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "vmtouch" %}
+{% set version = "1.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/hoytech/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: d57b7b3ae1146c4516429ab7d6db6f2122401db814ddd9cdaad10980e9c8428c
+
+build:
+  skip: true  # [win]
+  number: 0
+  script:
+    - make
```

You could use selectors to conditionally run `make install` here:
```suggestion
    - make
    - make install  # [not osx]
```
The macOS specific lines below can then be appended with `  # [osx]`.